PhysicsTools/NanoAOD/python/muons_cff.py

```diff
@@ -145,6 +166,7 @@
     externalVariables = cms.PSet(
         mvaTTH = ExtVar(cms.InputTag("muonMVATTH"),float, doc="TTH MVA lepton ID score",precision=14),
         mvaLowPt = ExtVar(cms.InputTag("muonMVALowPt"),float, doc="Low pt muon ID score",precision=14),
+        fsrPhotonIdx = ExtVar(cms.InputTag("muonFSRProducer:fsrIndex"),int, doc="Index of the associated FSR photon"),
```

Comment:
unit8_t?

Reply:
nope, it is -1 if no fsr photon is attached... as usual with indices

Reply:
unchanged